test/heex_formatter_test.exs

```diff
@@ -328,4 +328,24 @@ defmodule HeexFormatterTest do
 
     assert_formatter_output(input, expected)
   end
+
+  test "handle live_component format" do
```

Comment:
@paulo-silva let's add one more test covering forms. I think that will be a nice example to see how it formats 🙌🏻